pipeline.yml

```diff
@@ -7,14 +7,14 @@ jobs:
   - task: run-compliance-masonry
     config:
       platform: linux
-      image: docker:///geramirez/compliance-masonry-cli-build#0.4
+      image: docker:///geramirez/compliance-masonry-go#0.2
       inputs:
       - name: compliance-repository
       outputs:
       - name: masonry-exports
       run:
-        path: bash
-        args: ["-c", "masonry certs -d compliance-repository/data LATO && masonry docs -d compliance-repository/data -o masonry-exports gitbook LATO"]
+        path: "/bin/sh"
+        args: ["-c", "cd compliance-repository && compliance-masonry-go --verbose get && compliance-masonry-go --verbose docs gitbook -e ../masonry-exports FedRAMP-med"]  
```

Comment:
I'd switch this to be multi-line, so that it's easier to read. See the example here:

http://concourse.ci/running-tasks.html#section_task-anatomy

Ditto below.